addons/web_editor/static/src/js/wysiwyg/widgets/media.js

```diff
@@ -276,6 +277,9 @@ var ImageWidget = MediaWidget.extend({
         } else {
             domain = domain.concat(attachedDocumentDomain);
         }
+        if (this.options.document) {
+            domain.push('&', ['name', 'not like', 'assets_']);
```

Comment:
I don't see how this could work... if `domain` is `['|', A, B]`, after this line it will become `['|', A, B, '&', C]`, which is not a valid domain... is this working ? Try in cases where there are documents in the database

Anyways, try to share the domain targeting assets (where you found it for the regenerate button) and use it here with a correct combination of domains :wink: Maybe in the utils.js / misc.js file in web if no better option.